src/app/content/practiceQuestions/reducer.ts

```diff
@@ -39,13 +39,22 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action): State
         selectedSection: null,
       };
     case getType(actions.receivePracticeQuestionsSummary):
-      return {...state, summary: action.payload};
+      return { ...state, summary: action.payload };
     case getType(actions.setSelectedSection):
-      return {...state, selectedSection: action.payload, currentQuestionIndex: null, questions: []};
+      return {
+        ...state,
+        currentQuestionIndex: null,
+        questionAnswers: {},
+        questions: [],
+        selectedSection: action.payload,
+      };
     case getType(actions.nextQuestion):
-      return {...state, currentQuestionIndex: state.currentQuestionIndex === null ? 0 : state.currentQuestionIndex + 1};
+      return {
+        ...state,
+        currentQuestionIndex: state.currentQuestionIndex === null ? 0 : state.currentQuestionIndex + 1,
+      };
```

Comment:
this change is not necessary

Reply:
Its about the tslint max-line-length. Should I disable tslint error for such lines?

Reply:
error is there because you've added black spaces before/after curly brackets
if you remove them then there will be no error